skyreader/plot/plot.py

```diff
@@ -133,50 +139,79 @@ def create_plot(self, result: SkyScanResult, dozoom: bool = False) -> None:
             del this_map
             del grid_pix
 
-        LOGGER.info(f"min  RA: {min_ra *180./np.pi} deg, {min_ra*12./np.pi} hours.")
+        LOGGER.info(
+            f"min  RA: {min_ra * 180./np.pi} deg, {min_ra*12./np.pi} hours."
+        )
         LOGGER.info(f"min Dec: {min_dec * 180./np.pi} deg")
 
         # renormalize
         if dozoom:
             grid_map = grid_map - min_llh
-            # max_value = max_value - min_value
             min_llh = 0.
             max_llh = 50
-
+        if not llh_map:
+            prob_map = (copy.copy(grid_map) - min_llh)/2.
+            prob_map = -prob_map*np.log10(np.exp(1))
```

Comment:
I missed this in the earlier review. I don't see the smearing applied here and there's not a renormalization. Are you just flipping the delta-llh and calling that log(p)? That will be different from the other probability map I guess? I think instead we should either have a consistent definition of probability map, which maybe requires a function to do the conversion, or we can remove the `llh_map` option from this function.

Reply:
Yes, the idea was just to provide more or less an idea of the probability all over the sky. However yes, to plot exactly the same thing is definitely better. I created a new file in utils `handle_map_data.py`. Inside this file there are the functions to extract the map data from the output of skymap scanner and in case to immediately apply the convolution. In this way the map data can be obtained with the same function both by `create_plot` and `create_plot_zoomed` and what is plotted is exactly the same.